src/plugins/console/public/application/hooks/use_restore_request_from_history/restore_request_from_history.ts

```diff
@@ -32,7 +33,8 @@ export function restoreRequestFromHistory(editor: SenseEditor, req: ESRequest) {
 
   let s = prefix + req.method + ' ' + req.endpoint;
   if (req.data) {
-    s += '\n' + req.data;
+    const formattedData = formatRequestBodyDoc([req.data], true);
```

Comment:
what is this second argument passed as true? Could we define it as a const above this line and pass as param so that we can document what it does also? (this would also apply to the other file that we touched)

```typescript
const ARGUMENT_DESCRIPTION = true;
const formattedData = formatRequestBodyDoc([req.data], ARGUMENT_DESCRIPTION);
```